### ISSUE TITLE
Error: metadata.creationTimestamp is not nullable.

### ISSUE THREAD

**Issue by maikelvl** [fllaca/scheriff#12]
Hi, thanks for this awesome tool. ❤️ 

## Reproduction

I found an issue where `creationTimestamp: null` is found to be invalid.

```console
KUBERNETES_VERSION=1.17.11

curl -sL "https://raw.githubusercontent.com/kubernetes/kubernetes/v${KUBERNETES_VERSION}/api/openapi-spec/swagger.json" > k8s-$KUBERNETES_VERSION-openapi-specs.json

# Create a namespace
cat << EOF > test.namespace.yaml
apiVersion: v1
kind: Namespace
metadata:
  creationTimestamp: null
  name: test
EOF

scheriff -s k8s-${KUBERNETES_VERSION}-openapi-specs.json -f test.namespace.yaml
```

```
Validating config in test.namespace.yaml against schema in k8s-1.17.11-openapi-specs.json
Results:
Validating manifests in test.namespace.yaml:
	 - ERROR, test (v1/Namespace): Error at "/metadata/creationTimestamp":Value is not nullable
```

Is this a specification issue? Should we ignore this issue in scheriff?

## Temporary workaround

```
# Removing creationTimestamps before scheriff
find manifests -type f -name '*.yaml' | xargs -I{} sed "/^\s*creationTimestamp:\s*null$/d" -i {}
scheriff -s k8s-${KUBERNETES_VERSION}-openapi-specs.json -f test.namespace.yaml
```

**Comment by maikelvl** [comment]
Awesome! ❤️

**Comment by fllaca** [comment]
hi @maikelvl , thanks again for trying out scheriff and helping with this bug report! We just published v0.0.1-rc2 including the fix for this issue, here you have it: https://github.com/fllaca/scheriff/releases/tag/v0.0.1-rc2. I hope it serves you well!

I'd love to hear from you again, issues and PRs will be warmly welcome! ;D, cheers!
